pact.cabal

```diff
@@ -46,6 +46,11 @@ flag with-integer-gmp
   description: Include integer-gmp package (removed after GHC 9)
   manual:      False
 
+flag tests-in-lib
```

Comment:
Is this to workaround some cabal repl or ghci situation?

Reply:
Yes, it's to allow continuous development of tests without having to switch targets in `cabal repl`. Multi-homing will address this, but that's a ways away at this point.